### PR TITLE
no log passwords

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

 - Bugfix Pull Request

##### ANSIBLE VERSION

```
2.3
```

### CODE REVIEW

File: lib/ansible/modules/network/asa/asa_config.py [hunk 1]
```diff
@@ -311,7 +311,7 @@ def main():
 
         config=dict(),
         defaults=dict(type='bool', default=False),
-        passwords=dict(type='bool', default=False),
+        passwords=dict(type='bool', default=False, no_log=True),
```
Comment (gundalow) [inline lib/ansible/modules/network/asa/asa_config.py]: This doesn't look right. It's a bool, rather than a secret thing.

Comment (gundalow) [reply lib/ansible/modules/network/asa/asa_config.py]: This needs removing 
```
       - This argument specifies to include passwords in the config
         when retrieving the running-config from the remote device.  This
         includes passwords related to VPN endpoints.  This argument is
         mutually exclusive with I(defaults).
```

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/cloudstack/cs_host.py [hunk 2]
```diff
@@ -505,7 +505,7 @@ def main():
     argument_spec = cs_argument_spec()
     argument_spec.update(dict(
         name=dict(required=True, aliases=['url']),
-        password=dict(default=None, not_log=True),
```
Comment (resmo) [inline lib/ansible/modules/cloud/cloudstack/cs_host.py:508]: autch... bad typo.

File: lib/ansible/modules/network/asa/asa_config.py (hunk 1 above)
Comment (bcoca) [reply lib/ansible/modules/network/asa/asa_config.py]: fixed

**Review by gundalow — APPROVED** [review]

**Comment by sivel** [comment]
Just a somewhat related thought.  If we added a `password` type, we could have it imply `no_log`.

**Pull request merged**